suite/tests/api/ir_x86.c

```diff
@@ -1314,6 +1314,16 @@ test_xinst_create(void *dc)
     byte *pc;
     reg_id_t reg = DR_REG_XDX;
     instr_t *ins1, *ins2;
+    /* load 1 byte zextend */
```

Comment:
Thanks for adding the test.

Let's also add a line in the changelog in release.dox.